src/elements/video/demo/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html ng-app="demo">
+<head lang="en">
+  <meta charset="UTF-8">
+  <title></title>
+  <link rel="stylesheet" href="../../../../node_modules/semantic-ui-css/semantic.css"/>
+  <style>
+    body {
+      padding: 2em;
+    }
+  </style>
+</head>
+<body ng-controller="VideoDemoCtrl">
+
+  <div class="ui stackable centered grid">
+    <div class="row">
+      <div class="six wide column">
+        <h1 class="ui header">Youtube</h1>
+        <sm-video source="youtube" data-id="O6ROCaLvMnQ" placeholder="http://placekitten.com/601/300" pr-theme="light" options="options"
+                  on-ready="onReady(player)" pr-enablejsapi="1"></sm-video>
+      </div>
+    </div>
+    <div class="row">
+      <div class="six wide column">
```

Comment:
I am not sure how we are going to approach when to show control buttons and if we want them at all as a player has its own native controls